extmod/webrepl/webrepl.py

```diff
@@ -39,13 +50,20 @@ def accept_conn(listen_sock):
         return
     print("\nWebREPL connection from:", remote_addr)
     client_s = cl
-    websocket_helper.server_handshake(cl)
+    if websslrepl:
+        if hasattr(uos, 'dupterm_notify'):
+            cl.setsockopt(socket.SOL_SOCKET, 20, uos.dupterm_notify)
+        cl = ssl.wrap_socket(cl, server_side=True, key=key, cert=cert)
+        websocket_helper.server_handshake(cl, ssl=True)
+    else:
+        websocket_helper.server_handshake(cl)
     ws = uwebsocket.websocket(cl, True)
     ws = _webrepl._webrepl(ws)
     cl.setblocking(False)
     # notify REPL on socket incoming data (ESP32/ESP8266-only)
```

Comment:
Ah, I guess this comment answers my question above... It would have been nice to keep the ssl/non-ssl code together with the comment...